hist/hist/test/test_TFormulaPol.cxx

```diff
@@ -0,0 +1,65 @@
+#include "gtest/gtest.h"
+#include "TFormula.h"
+#include "TF1.h"
+#include "ROOT/TestSupport.hxx"
+#include <cmath>
+
+using namespace ROOT;
+
+class TFormulaPolTest : public ::testing::Test {
+protected:
+    void SetUp() override {
+        TestSupport::CheckDiagsRAII diags;
+    }
+};
```

Comment:
This class is most likely not needed. You could remove it and use the following expression to instantiate a test:
`TEST(TFormula, xxx)`